pkg/trait/ingress.go

```diff
@@ -126,6 +128,15 @@ func (t *ingressTrait) Apply(e *Environment) error {
 		},
 	}
 
+	if len(t.TLSHosts) > 0 || len(strings.TrimSpace(t.TLSSecretName)) > 0 {
+		ingress.Spec.TLS = []networkingv1.IngressTLS{
+			{
+				Hosts:      t.TLSHosts,
+				SecretName: t.TLSSecretName,
+			},
+		}
+	}
```

Comment:
@squakez :
like I said, not a Go developper.
They may be a better way to do this ?

What we want, is to completely not include the ingress.Spec.TLS if TLSHosts or TLSSecretName is not set

Reply:
ok, doubled check with the doc, and yes, need both.
so I'll change to "and" and add more tests

Reply:
replace by 
len(t.TLSHosts) > 0 && t.TLSSecretName != ""
+ 2 test added to cover both part of the if